pkg/config/config.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+const (
+	InitConfigurationTemplate = `---
+{{.ClusterConfiguration}}
+---
+{{.InitConfiguration}}
+---
+{{.KubeletConfiguration}}
+`
+)
+
+const (
+	JoinConfigurationTemplate = `---
+{{.JoinConfiguration}}
+---
+{{.KubeletConfiguration}}
+`
+)
+
+type Init struct {
+	ClusterConfiguration string
```

Comment:
can we import these structs from the kubeadm package instead of trying to replicate them?